components/dashboard/src/components/UsageView.tsx

```diff
@@ -160,13 +179,61 @@ function UsageView({ attributionId }: UsageViewProps) {
 
     const headerTitle = attributionId.kind === "team" ? "Team Usage" : "Personal Usage";
 
+    const FromLink = forwardRef((arg: any, ref: any) => (
+        <div
+            className="text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            {arg.value}
+        </div>
+    ));
+    const ToLink = forwardRef((arg: any, ref: any) => (
+        <div
+            className="text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer"
+            onClick={arg.onClick}
+            ref={ref}
+        >
+            {arg.value}
+        </div>
+    ));
+
     return (
         <>
             <Header
-                title={headerTitle}
-                subtitle={`${new Date(startDateOfBillMonth).toLocaleDateString()} - ${new Date(
-                    endDateOfBillMonth,
-                ).toLocaleDateString()} (updated every 15 minutes).`}
+                title={
+                    <div className="flex items-baseline">
+                        <h1 className="tracking-wide">{headerTitle}</h1>
+                        <h2 className="ml-3">(updated every 15 minutes).</h2>
+                    </div>
+                }
+                subtitle={
+                    <h2 className="tracking-wide flex mt-3">
+                        <h2 className="mr-2">Period: </h2>
+                        {startDateError}
```

Comment:
**issue**: Probably not the same error as in the screenshot, and maybe out of the scope of the changes, but we should probably impose a limit on the the max date range gap like 180 days or something, right? A good MVC could be to surface a danger alert with the error message instead of breaking the page.

<img width="1131" alt="Screenshot 2022-11-09 at 9 39 34 PM" src="https://user-images.githubusercontent.com/120486/200925693-9e80e26d-eca2-4fbd-bfd8-7d1e550764f0.png">